client/my-sites/subscribers/components/subscriber-list-container/subscriber-list-container.tsx

```diff
@@ -1,4 +1,5 @@
-import { numberFormat, translate } from 'i18n-calypso';
+import formatNumber from '@automattic/components/src/number-formatters/lib/format-number';
```

Comment:
I somehow remember this had issues with different locales (try Spanish), and didn't fully work as expected on all number fractions. I could be wrong!

Reply:
I will do some testing, thanks!

Reply:
I tested it and it seems it works fine.

Added a change to use a user locale instead of the browser one.

Reply:
I think I mixed with `numberFormat` like you noted on Slack 👍  That one was problematic indeed.